controller/src/main/java/io/pravega/controller/fault/FailoverSweeper.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.controller.fault;
+
+import java.util.Set;
+import java.util.concurrent.CompletableFuture;
+import java.util.function.Supplier;
+
```

Comment:
Please enter a short description at the top of this class to say what a failover sweeper is supposed to do, what kind of specialization we expect here, etc.